From a01fee0ef04e93dbe1eecb1aa276e515114e9c1b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Mon, 14 Oct 2013 19:15:24 -0400
Subject: [PATCH] Fix bad assert in show_pid_array

This function should get the same treatment as other qsort uses
did in 7ff7394 "Never call qsort on potentially NULL arrays".

Reported-by: Oleksii Shevchuk <alxchk@gmail.com>
---
 src/shared/cgroup-show.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/shared/cgroup-show.c b/src/shared/cgroup-show.c
index f6e56db..aa0f017 100644
--- a/src/shared/cgroup-show.c
+++ b/src/shared/cgroup-show.c
@@ -44,8 +44,6 @@ static void show_pid_array(int pids[], unsigned n_pids, const char *prefix, unsi
         unsigned i, m, pid_width;
         pid_t biggest = 0;
 
-        assert(n_pids > 0);
-
         /* Filter duplicates */
         m = 0;
         for (i = 0; i < n_pids; i++) {
